src/main/scala/com/databricks/labs/overwatch/MultiWorkspaceDeployment.scala

```diff
@@ -166,7 +166,8 @@ class MultiWorkspaceDeployment extends SparkSessionWrapper {
       config.enable_unsafe_SSL.getOrElse(false),
       config.thread_pool_size.getOrElse(4),
       config.api_waiting_time.getOrElse(300000),
-      Some(apiProxyConfig))
+      Some(apiProxyConfig),
+      Some(config.mount_mapping_path))
```

Comment:
Did you add some validation for this? When this parameter is passed, the >50 mount validation should be disabled and the mapping validation should be enabled. Is this done?

Reply:
no, i thought of it but some part of the code is in 
https://github.com/databrickslabs/overwatch/pull/713
and some part of the code is in
https://github.com/databrickslabs/overwatch/pull/695

so was waiting for these 2 PR to get merged so that i can enhance it to add the validation part

Reply:
Ok, will merge this now and #713 as soon as your review is complete, and you can add the validator logic comments in a new PR after all merged. Thanks.